tmail-backend/apps/distributed-es6-backport/pom.xml

```diff
@@ -414,7 +414,21 @@
                             <port>8000</port> <!-- WEBADMIN -->
                         </ports>
                         <appRoot>/root</appRoot>
+                        <jvmFlags>
+                            <jvmFlag>-Dlogback.configurationFile=/root/conf/logback.xml</jvmFlag>
+                            <jvmFlag>-Dworking.directory=/root/</jvmFlag>
+                            <!-- Prevents Logjam (CVE-2015-4000) -->
+                            <jvmFlag>-Djdk.tls.ephemeralDHKeySize=2048</jvmFlag>
+                            <jvmFlag>-Dextra.props=/root/conf/jvm.properties</jvmFlag>
```

Comment:
Can you try adding this:
```
                                <jvmFlag>--add-opens=java.base/java.lang=ALL-UNNAMED</jvmFlag>
                                <jvmFlag>--add-opens=java.base/java.math=ALL-UNNAMED</jvmFlag>
                                <jvmFlag>--add-opens=java.base/java.util=ALL-UNNAMED</jvmFlag>
                                <jvmFlag>--add-opens=java.base/java.util.concurrent=ALL-UNNAMED</jvmFlag>
                                <jvmFlag>--add-opens=java.base/java.net=ALL-UNNAMED</jvmFlag>
                                <jvmFlag>--add-opens=java.base/java.text=ALL-UNNAMED</jvmFlag>
```

Reply:
yes good point... actually it's that commit that is the problem. I don't know why you need the flags declared here for that too but it's a good remark. But by just kicking this out it works as well. So I reopened https://github.com/linagora/tmail-backend/pull/469 where I will apply your suggestion instead, to speed things up. 

But thanks for taking a look :)

Reply:
>  I don't know why you need the flags declared here for that too but it's a good remark

I think somehow now rabbitmq client production code call some reflection code that is not enabled by default on Java 17. So actually our product' JRE needs to support that reflection, not just in the maven test JRE.